auth/login.html

```diff
@@ -1,15 +1,21 @@
 {{define "content"}}
-	<h2>Login</h2>
-	<form method="post">
-		<div class="form-group">
-			<label for="username">User</label>
-			<input class="form-control" type="text" id="username" name="username" />
+<div class="container">
+		<h1 class="welcome">Reach your community</h1>
+		<h1 class="welcome">with <span class="luckdrum">LuckDrum</span></h1>
+		<p class="desc">Earth's GLOCAL broadcasting drum and message wall</p>
+		<div class="login-container col-md-4">
+			<h3>Log in</h3>	
```

Comment:
I'm skipping from ```h1``` to ```h3```, because ```h2``` is too large for this field. 
Redesign it isn't worth it.